src/devtools/client/debugger/src/actions/ui.js

```diff
@@ -23,6 +24,8 @@ import { isFulfilled } from "../utils/async-value";
 
 import { getCodeMirror } from "devtools/client/debugger/src/utils/editor";
 import { resizeBreakpointGutter } from "../utils/ui";
+import { prefs } from "../utils/prefs";
+import { getSelectedPrimaryPanel } from "../../../../../ui/reducers/app";
```

Comment:
Same here, if we can use `import { getSelectedPrimaryPanel } from "ui/reducers";` it would be cleaner. I'd try that if it works.